x-pack/plugins/security_solution/server/endpoint/routes/actions/audit_log_handler.ts

```diff
@@ -53,7 +78,10 @@ export const actionsLogRequestHandler = (
     }
 
     return res.ok({
-      body: result.body.hits.hits.map((e) => e._source),
+      body: result.body.hits.hits.map((e) => {
```

Comment:
I think it would be interesting to return the hits.total and not modify at all the elastic response.
What about adding the type field inside each document but still returning the ES response itself?

Reply:
Yeah not quite sure what you mean, but I can see that the `total` field to be some constant value on every request, this doesn't equate with the number of records in the result. Besides, we are interested in the data array within the `hits.hits._source` key. At least that's how fleet indices return the data.

I would however like a better way to distinguish the indices instead of using a regex, but as I can see fleet responds with index names with prefixes and suffixes, instead of just `.fleet-actions` and `.fleet-actions-results`.

![Screenshot 2021-06-08 at 15 24 26](https://user-images.githubusercontent.com/1849116/121192846-9dd67480-c86d-11eb-9554-497ae71a941f.png)

Reply:
This is a generic ES response:
```
{
  "took" : 610,
  "timed_out" : false,
  "_shards" : {
    "total" : 4,
    "successful" : 4,
    "skipped" : 0,
    "failed" : 0
  },
  "hits" : {
    "total" : {
      "value" : 1151,
      "relation" : "eq"
    },
    "max_score" : 1.0,
    "hits" : []
}
}
```
`hits.total` will give you the total of results in the index by query. Sometimes this is useful to know if you can paginate more, to display the total number of records on the UI, etc.

I would suggest to keep this response format and just adding the type on each document.

Reply:
you almost definitely don't need regexes here. just string prefixes.

` _index.startsWith(".fleet-actions") ` === "action" type

` _index.startsWith(".ds-.fleet-actions-results")` === "result"

They should all fit that pattern, no?

Reply:
done 4f7d18bee78cdc389ab0bf3cb94271c66b4ae25b